Terminal.Gui/Drawing/ColorEventArgs.cs

```diff
@@ -0,0 +1,16 @@
+﻿#nullable enable
+
+namespace Terminal.Gui;
+
+/// <summary>Event arguments for the <see cref="Color"/> events.</summary>
+public class ColorEventArgs : EventArgs
```

Comment:
For consistency, it would be nice if this followed the pattern I introduced in IOrientation.

For non-canceable use `EventArgs<Color>`.  For cancelable use `CancelEventArgs<Color>`

Reply:
Done in ea614953dd89b69db01963cd91bc222b325791db